packaging/windows/khiops_env.cmd

```diff
@@ -0,0 +1,156 @@
+echo   . set to 'false' to allow local file name selection only with a file selection dialog
+echo.
+echo In case of configuration problems, the variables KHIOPS_JAVA_ERROR and KHIOPS_MPI_ERROR contain error messages.
+
+if not %2.==. exit /b 1
+if %1.==-h. exit /b 0
+if %1.==--help. exit /b 0
+exit /b 1
+
+REM Set Khiops environment variables
+:DISPLAY_ENV
+setlocal
+set DISPLAY_ENV=true
+
+:SET_ENV
+REM Initialize exported variables
+set "KHIOPS_PATH="
+set "KHIOPS_COCLUSTERING_PATH="
+set "KHIOPS_MPI_COMMAND="
+set "KHIOPS_JAVA_PATH="
+set "KHIOPS_CLASSPATH="
+set "KHIOPS_JAVA_ERROR="
+set "KHIOPS_MPI_ERROR="
+
+REM Set Khiops home to parent directory
+for %%a in ("%~dp0..") do set "_KHIOPS_HOME=%%~fa"
+
+REM KHIOPS_PATH
+set "KHIOPS_PATH=%_KHIOPS_HOME%\bin\MODL.exe"
+set "KHIOPS_COCLUSTERING_PATH=%_KHIOPS_HOME%\bin\MODL_Coclustering.exe"
+
+REM KHIOPS_LAST_RUN_DIR
+if "%KHIOPS_LAST_RUN_DIR%".=="". set "KHIOPS_LAST_RUN_DIR=%USERPROFILE%\khiops_data\lastrun"
+
+REM KHIOPS_PROC_NUMBER
+if "%KHIOPS_PROC_NUMBER%".=="". for /f %%i in ('"%_KHIOPS_HOME%"\_getprocnumber') do set "KHIOPS_PROC_NUMBER=%%i"
+if "%KHIOPS_PROC_NUMBER%".=="". set "KHIOPS_PROC_NUMBER=1"
+
+REM set MPI environment
+if %KHIOPS_PROC_NUMBER% EQU 1 goto MPI_DONE
+REM ... with the standard variable MSMPI_BIN
+set "KHIOPS_MPI_COMMAND=%MSMPI_BIN%mpiexec.exe"
+REM ... if MSMPI_BIN is not correctly defined 
+REM ... we try to call directly mpiexec (assuming its path is in the 'path' variable)
+if not exist "%KHIOPS_MPI_COMMAND%" goto USE_MPI_IN_PATH
+:MPI_PARAMS
+REM ... we add the MPI parameters and the process priority to 1 (0:Idle, 1: Below normal, 2: Normal)
+if not "%KHIOPS_MPI_COMMAND%."=="." set "KHIOPS_MPI_COMMAND="%KHIOPS_MPI_COMMAND%" -al spr:P -n %KHIOPS_PROC_NUMBER% /priority 1"
+if %KHIOPS_PROC_NUMBER% EQU 1 set "KHIOPS_MPI_COMMAND="
+:MPI_DONE
+
+set _KHIOPS_GUI=@GUI_STATUS@
+if "%_KHIOPS_GUI%" == "false" GOTO SKIP_GUI
+
+REM Set Java environment
+set _JAVA_ERROR=false
+if not exist "%_KHIOPS_HOME%\jre\bin\server\" set _JAVA_ERROR=true
+if not exist "%_KHIOPS_HOME%\jre\bin\" set _JAVA_ERROR=true
+
+if  "%_JAVA_ERROR%" == "false" (
+    set "KHIOPS_JAVA_PATH=%_KHIOPS_HOME%\jre\bin\server\;%_KHIOPS_HOME%\jre\bin\"
+) else set "KHIOPS_JAVA_ERROR=The JRE is missing in Khiops home directory, please reinstall Khiops"
+
+REM KHIOPS_CLASSPATH
+set "KHIOPS_CLASSPATH=%_KHIOPS_HOME%\bin\norm.jar"
+set "KHIOPS_CLASSPATH=%_KHIOPS_HOME%\bin\khiops.jar;%KHIOPS_CLASSPATH%"
+
+:SKIP_GUI
+
+REM unset local variables
+set "_KHIOPS_GUI="
+set "_JAVA_ERROR="
+set "_KHIOPS_HOME="
+
+if not "%DISPLAY_ENV%" == "true" exit /b 0
+
+REM Print the environment list on the standard output
+echo KHIOPS_PATH %KHIOPS_PATH%
+echo KHIOPS_COCLUSTERING_PATH %KHIOPS_COCLUSTERING_PATH%
+echo KHIOPS_MPI_COMMAND %KHIOPS_MPI_COMMAND%
+echo KHIOPS_JAVA_PATH %KHIOPS_JAVA_PATH%
+echo KHIOPS_CLASSPATH %KHIOPS_CLASSPATH%
+echo KHIOPS_LAST_RUN_DIR %KHIOPS_LAST_RUN_DIR%
```

Comment:
Shouldn't we enclose "%KHIOPS_PATH%" etc between quotes, so that they are not truncated as spaces?